TeX.gitignore

```diff
@@ -279,3 +282,7 @@ TSWLatexianTemp*
 # option is specified. Footnotes are the stored in a file with suffix Notes.bib.
 # Uncomment the next line to have this generated file ignored.
 #*Notes.bib
+
+# Sublime Text
```

Comment:
We maintain a separate [`Global/SublimeText.gitignore`](https://github.com/github/gitignore/blob/master/Global/SublimeText.gitignore) template containing these rules. Please drop these changes from the template.

Reply:
will close this pull request and submit a new one